lib/install/examples/stimulus/controllers/hello_controller.js

```diff
@@ -0,0 +1,12 @@
+// Visit The Stimulus Handbook for more details 
+// https://stimulusjs.org/handbook/introduction
+
+import { Controller } from "stimulus"
+
+export default class extends Controller {
+  static targets = [ "output" ]
+  connect() {
+    this.outputTarget.textContent =
+    `Hello Stimulus!`
```

Comment:
No need for the `\n` here or the interpolated string: `this.outputTarget.textContent = 'Hello Stimulus!'`